recipe/meta.yaml

```diff
@@ -259,8 +259,8 @@ outputs:
         - python
         - {{ pin_subpackage('ray-default', exact=True) }}
         - pandas
+        - pyarrow >=6.0.1
```

Comment:
Is it correct that ray-rllib has gained a pyarrow-dependency?

I don't see that in the "remove all dependencies" patch, but you added it when updating the requirements - C&P gone astray perhaps?

Reply:
Ah, sorry, I had an issue with expanding the file correctly in the GH UI apparently, I see now that this is for `ray-tune`. Apologies.